kontrakter/siktelseTiltale/1.0/siktelseTiltale.schema.json

```diff
@@ -0,0 +1,870 @@
+            "kravId": {
+              "$ref": "#/definitions/unikId"
+            }
+          },
+          "additionalProperties": false,
+          "required": ["kravId"]
+        },
+        {
+          "properties": {
+            "anmodningsId": {
+              "$ref": "#/definitions/unikId"
+            }
+          },
+          "additionalProperties": false,
+          "required": ["anmodningsId"]
+        }
+      ]
+    },
+    "adresse": {
+      "type": "object",
+      "description": "Norsk eller utenlandsk adresse",
+      "properties": {
+        "norge": {
+          "$ref": "#/definitions/adresseNorge",
+          "description": "Norsk adresse"
+        },
+        "utland": {
+          "$ref": "#/definitions/adresseUtland"
+        }
+      },
+      "oneOf": [
+        {
+          "properties": {
+            "norge": {
+              "$ref": "#/definitions/adresseNorge"
+            }
+          },
+          "additionalProperties": false,
+          "required": ["norge"]
+        },
+        {
+          "properties": {
+            "utland": {
+              "$ref": "#/definitions/adresseUtland"
+            }
+          },
+          "additionalProperties": false,
+          "required": ["utland"]
+        }
+      ]
+    },
+    "adresseNorge": {
+      "type": "object",
+      "description": "Norske med litt strengere validering på postnummer",
+      "properties": {
+        "adresselinjer": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 500
+          },
+          "maxItems": 4
+        },
+        "postnummer": {
+          "$ref": "#/definitions/norskPostnummer"
+        },
+        "poststed": {
+          "type": "string"
+        }
+      },
+      "required": ["adresselinjer", "postnummer"],
+      "additionalProperties": false
+    },
+    "adresseUtland": {
+      "type": "object",
+      "properties": {
+        "adresselinjer": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 500
+          },
+          "minItems": 1,
+          "maxItems": 4
+        },
+        "postnummer": {
+          "type": "string"
+        },
+        "poststed": {
+          "type": "string"
+        },
+        "land": {
+          "$ref": "#/definitions/land"
+        }
+      },
+      "required": ["adresselinjer"],
+      "additionalProperties": false
+    },
+    "kodeverk": {
+      "type": "object",
+      "properties": {
+        "kode": {
+          "type": "string"
+        },
+        "navn": {
+          "type": "string"
+        }
+      },
+      "required": ["kode"],
+      "additionalProperties": false
+    },
+    "kontaktInfoPerson": {
+      "type": "object",
+      "description": "Alle har epost adresse, men kanskje ikke registrert mobil/telefon",
+      "properties": {
+        "epost": {
+          "type": "string"
```

Comment:
vi kunne vurdert egen type for **epost**. Gjerne med pattern-matching også.
Har ikke brukt det selv, men vurderer å legge inn denne: `^[^\s@]+@([^\s@.,]+\.)+[^\s@.,]{2,}$`

Reply:
ok